mypy/checkexpr.py

```diff
@@ -200,8 +200,14 @@ def analyze_var_ref(self, var: Var, context: Context) -> Type:
     def visit_call_expr(self, e: CallExpr, allow_none_return: bool = False) -> Type:
         """Type check a call expression."""
         if e.analyzed:
+            if isinstance(e.analyzed, NamedTupleExpr) and not e.analyzed.is_typed:
+                # Type check the arguments, but ignore the results.
```

Comment:
Could you please add a comment here that generating correct errors in call here depends on the typeshed stub?